firebase/firebase-cpp-sdk
# Feature/tvos support integration tests firestore internal

Changes done to XCode project for firestore internal integration tests that allows us to run integration tests on device and simulator for tvos.

added a tvos target to XCode project for firestore internal integration tests.
updated Podfile to build pods for tvos platform for this newly added target
To test this,

Make sure to have a valid GoogleService-Info.plist.
pod install
open integration_test.xcworkspace
Drag and drop firebase.xcframework, firebase_auth.xcframework and firebase_firestore.xcframework.

All tests pass on simulator and device. 

Notes: needed an additional step to copy over the preprocessor macro defines from ios target to tvos target.
